armi/reactor/converters/uniformMesh.py

```diff
@@ -115,7 +115,9 @@ def convert(self, r=None):
         self._setParamsToUpdate()
         self._computeAverageAxialMesh()
         self._buildAllUniformAssemblies()
-        self._mapStateFromReactorToOther(self._sourceReactor, self.convReactor)
+        self._mapStateFromReactorToOther(
+            self._sourceReactor, self.convReactor, mapNumberDensities=False
```

Comment:
Number densities are mapped during `_buildAllUniformAssemblies`; don't need to do it here.